### ISSUE TITLE
Docker container does not have pandoc installed

### ISSUE THREAD

**Issue by user-name-is-taken** [DannyBen/bashly#477]
### Bashly Version

Latest Docker Image

### Description

The docker image ID `19b81be004c4` does not contain pandoc.

### Reproduction Steps

```
$ bashly render :mandoc docs
```

### Actual Behavior

```
saved docs/chrootdisk.md
sh: pandoc: not found
 RuntimeError 
Failed running pandoc
Make sure the following command succeeds and try again:

  pandoc -f markdown-smart -s --to man "docs/chrootdisk.md" > "docs/chrootdisk.1"
```

### Expected Behavior

Generate a man page successfully

**Comment by DannyBen** [comment]
You are absolutely right. Thanks for the report, I will fix it today I hope.

**Comment by DannyBen** [comment]
Fixed in #478 and pushed a new image. Try now.
